spectral_cube/spectral_cube.py

```diff
@@ -1861,7 +1861,7 @@ def subcube_from_ds9region(self, ds9region, allow_empty=False):
             ylo = 0
 
         log.debug("Region boundaries: ")
-        log.debug("xlo={xlo}, ylo={ylo}, xhi={xhi}, yhi={yhi}".format(xlo=xlo,
+        print("xlo={xlo}, ylo={ylo}, xhi={xhi}, yhi={yhi}".format(xlo=xlo,
```

Comment:
Instead of turning this into a print statement, you can do:
```
from astropy import log
log.setLevel('DEBUG')
```
to enable debug statement prints